radix-engine/src/engine/kernel.rs

```diff
@@ -1158,7 +1121,7 @@ where
         // TODO: Authorization
 
         // Take any required child nodes
-        let children = re_node.get_child_nodes()?;
+        let children = re_node.get_loaded_child_nodes()?;
```

Comment:
Why `loaded_child_nodes`? Feels wrong as this would mean that only loaded ones are taken?

Reply:
Renamed to the original version and added note that it should NOT be used for tracked nodes.

Reply:
Still looks like the new version? :)

Reply:
Don't know what happened to my change. :(

Will add again.